.github/workflows/codecov.yml

```diff
@@ -0,0 +1,33 @@
+name: codecov
+on:
+  push:
+    branches: [main, 0.4.x]
+  pull_request:
+jobs:
+  # Run code coverage using cargo-llvm-cov then upload to codecov.io
+  job_code_coverage:
+    name: llvm-cov
+    runs-on: ubuntu-latest
+    env:
+      COVERAGE_FILE: ./coverage.lcov
+    steps:
+      - uses: taiki-e/install-action@cargo-llvm-cov
```

Comment:
I'd like this to stick as closely as possible to the GH Actions example in the cargo-llvm-cov README, or have comments that explain any required differences.